controllers/quiz_controller.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const db = require("../models");
+const passport = require("../config/passport");
+const axios = require("axios");
+
+const router = express.Router();
+
+router.get("/quiz/15", (req, res) => {
+  db.Question.findAll({
+    include: [db.Answer]
+  }).then(dbQuestion => {
+    const resultsJSON = dbQuestion.map(result => {
+      return result.toJSON();
+    });
+
+    // console.log("\x1b[33m%s\x1b[0m", resultsJSON); //yellow
+    res.render("quiz", { trivia: resultsJSON });
+  });
+
+  //   const quizURL = `https://opentdb.com/api.php?amount=10&category=${req.params.id}&difficulty=easy&type=multiple`;
+
+  //   axios.get(quizURL).then(results => {
+  //     console.log("results: ", results);
+});
+
+router.get("/api/categories", (req, res) => {
+  db.Category.findAll({}).then(dbCategory => {
+    res.json(dbCategory);
+  });
+});
+
+router.post("/api/newScore", (req, res) => {
+  db.Score.create(req.body).then(dbScore => {
+    res.json(dbScore);
+  });
+});
+
+router.get("/api/scores", (req, res) => {
+  db.Score.findAll({
+    include: [db.Category, db.User]
+  }).then(dbScore => {
+    res.json(dbScore);
+  });
+});
+g;
```

Comment:
you have an extra g here, your build should have failed... make sure the code you push works... not sure important for this project but this is one of the things for which there is little leniency in the real world (pushing code that doesn't build/run)